common/src/test/java/tachyon/worker/block/io/LocalFileBlockReaderTest.java

```diff
@@ -28,17 +28,31 @@
 
 import tachyon.util.io.BufferUtils;
 
+/**
+ * Tests for the {@link LocalFileBlockReader} class.
+ */
 public class LocalFileBlockReaderTest {
   private static final long TEST_BLOCK_SIZE = 1024;
   private LocalFileBlockReader mReader;
   private String mTestFilePath;
 
+  /**
+   * Rule for the temporary folder.
```

Comment:
Can we put field-level comments on one line if they fit, e.g.
`/** Rule to create a new temporary folder during each test */`